apps/teams-test-app/public/manifest.json

```diff
@@ -1,10 +1,52 @@
 {
-  "short_name": "Teams Test App",
-  "name": "Teams Test App",
-  "icons": [
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+  "version": "1.0.0",
+  "manifestVersion": "1.16",
+  "id": "a79a2374-ec2a-486e-a3dc-b913fdb1f365",
+  "packageName": "com.package.name",
+  "name": { "short": "M365 Test App", "full": "" },
+  "developer": {
+    "name": "M365",
+    "mpnId": "",
+    "websiteUrl": "https://<REPLACE WITH URL OF THE APP>",
+    "privacyUrl": "https://<REPLACE WITH URL OF THE APP>",
+    "termsOfUseUrl": "https://<REPLACE WITH URL OF THE APP>"
+  },
+  "description": {
+    "short": "App to test infrastructure",
+    "full": "App to test infrastructure"
+  },
+  "icons": { "outline": "outline.png", "color": "color.png" },
+  "accentColor": "#FFFFFF",
+  "configurableTabs": [
+    {
+      "configurationUrl": "https://<REPLACE WITH URL OF THE APP>",
+      "canUpdateConfiguration": true,
+      "scopes": ["groupChat", "team"],
+      "context": [
+        "channelTab",
+        "privateChatTab",
+        "meetingSidePanel",
+        "meetingStage",
+        "meetingDetailsTab",
+        "meetingChatTab"
+      ]
+    }
   ],
-  "start_url": "/",
-  "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+  "staticTabs": [
+    {
+      "entityId": "test",
+      "name": "Test",
+      "contentUrl": "https://<REPLACE WITH URL OF THE APP>",
```

Comment:
Why not list the actual endpoint that you guys are hosting this app at?

Reply:
Great question! We actually don't currently deploy to that endpoint anymore. It's still there, but hasn't been updated in quite a while. Debating whether to do the work to re-enable it, but until then it's likely that people will want to point this at a locally run version of the app.